docs/buildstrategies.md

```diff
@@ -106,6 +110,31 @@ kubectl apply -f samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml
 
 ---
 
+## Buildkit
+
+[Buildkit](https://github.com/moby/buildkit) is composed of the `buildctl` client and the `buildkitd` daemon. For the `buildkit` ClusterBuildStrategy, it runs on a [daemonless](https://github.com/moby/buildkit#daemonless) mode, where both client and ephemeral daemon run in a single container. In addition, it runs without privileges ( _[rootless](https://github.com/moby/buildkit/blob/master/docs/rootless.md)_ ).
+
+### Cache Exporters
+
+By default, the `buildkit` ClusterBuildStrategy will use caching to optimize the build times. When pushing an image to a registry, it will use the `inline` export cache, which pushes the image and cache together.
```

Comment:
Is it worth mentioning how the cache is stored in the container registry? Maybe a link to the buildkit documentation on that part (https://github.com/moby/buildkit#export-cache, inline)

Reply:
indeed, done